.github/workflows/e2e810.yaml

```diff
@@ -6,8 +6,8 @@ on:
   workflow_dispatch:
 jobs:
   e2e-tests:
-    runs-on: self-hosted
-    if: contains(github.event.label.name, 'e2e-test')
+    if: contains(github.event.label.name, 'e2e-test') || (github.event_name == 'workflow_dispatch')
+    runs-on: [self-hosted, 810]
```

Comment:
Is [...] a OR or AND logic? If it is OR logic, Shouldn't this be: [810]?

Reply:
It is AND logic. Omitting self-hosted label will work but it is not recommended in Github documentation.

Reply:
Just confirming what Andrew said
From: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on
"If you specify an array of strings, your workflow will execute on any runner that matches **all** of the specified runs-on values."